spec/fixtures/pyproject.toml

```diff
@@ -8,9 +8,14 @@ authors = ["Tyrel Souza <tyrel@tidelift.com>"]
 python = "^3.7"
 django = "^3.0.7"
 
+# Old way to define dev deps (<1.2.0)
 [tool.poetry.dev-dependencies]
 pytest = "^5.2"
 
+# New way to define dev deps (>=1.2.0)
+[tool.poetry.group.dev.dependencies]
```

Comment:
We should add another group here and test that the type works correctly. Something like `[tool.poetry.group.test.dependencies]`.